sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -3268,18 +3272,22 @@ def __init__(self, **kwargs):
         self.last_updated_on = kwargs.get("last_updated_on", None)
         self.kind = kwargs.get("kind", None)
         self.resource_location = kwargs.get("resource_location", None)
+        self.api_version = kwargs.get("api_version", None)
+        self.tags = kwargs.get("tags", None)
 
     def __repr__(self):
         return (
             "ModelOperationInfo(operation_id={}, status={}, percent_completed={}, created_on={}, last_updated_on={}, "
-            "kind={}, resource_location={})".format(
+            "kind={}, resource_location={}, api_version={}, tags={})".format(
                 self.operation_id,
                 self.status,
                 self.percent_completed,
                 self.created_on,
                 self.last_updated_on,
                 self.kind,
                 self.resource_location,
+                self.api_version,
+                repr(self.tags),
```

Comment:
I don't think it matters a ton, but I don't think we need the repr here around a standard dict